src/olympia/zadmin/tasks.py

```diff
@@ -615,8 +615,9 @@ def fetch_langpack(url, xpi, **kw):
         sign_file(file_, settings.SIGNING_SERVER)
 
         # Finally, set the addon summary if one wasn't provided in the xpi.
-        addon.update(status=amo.STATUS_PUBLIC,
-                     summary=(addon.summary if addon.summary else addon.name))
+        addon.status = amo.STATUS_PUBLIC
+        addon.summary = addon.summary if addon.summary else unicode(addon.name)
+        addon.save(update_fields=('status', 'summary'))
```

Comment:
I'm scratching my head why this does something different than the `.update` statement but I believe you